lib/clerk/rack_middleware_v2.rb

```diff
@@ -231,6 +234,8 @@ def verify_token(token)
 
       begin
         sdk.verify_token(token)
+      rescue JWT::ExpiredSignature, JWT::InvalidIatError => e
+        raise e
```

Comment:
What is the purpose of these lines? They seem like redundant to me.

Reply:
Since `JWT::DecodeError` is the parent class of all `JWT::<Error>` i added this line of code (before the `rescue JWT::DecodeError`) to handle those 2 cases using interstitial by re-raising the error and letting the invoking handle it.
In the `call` method that `verify_token` is used i rescue those errors and show interstitial.